authentication/models.py

```diff
@@ -52,6 +53,36 @@ class UserBank(models.Model):
     item_id = models.CharField(max_length=1000)
     access_token = models.CharField(max_length=1000)
     institution_name = models.CharField(max_length=1000)
+    current_balance = models.DecimalField(max_digits=15, decimal_places=2)
+    account_name = models.CharField(max_length=1000)
+    income = models.DecimalField(max_digits=15, decimal_places=2)
+    expenditure = models.DecimalField(max_digits=15, decimal_places=2)
+
+
+    def current_balance(self, update=False):
+        if update or self.current_balance is None:
+            self.current_balance = request.plaid.current_balance()
```

Comment:
Instead of request.plaid make a method _plaid_client(self) that returns a plaid client.